deployment/terraform/dns.tf

```diff
@@ -0,0 +1,14 @@
+#
+# Public DNS resources
+#
+resource "aws_route53_record" "api" {
+  zone_id = data.terraform_remote_state.core.outputs.route53_public_hosted_zone_id
```

Comment:
I believe that we'll have to take this in as a Terraform variable as well. We may be able to support using a `data` resource that looks things up within the account by domain name too.